Assets/HoloToolkit/BuildAndDeploy/Editor/BuildDeployTools.cs

```diff
@@ -80,13 +80,11 @@ public static string CalcMSBuildPath(string msBuildVersion)
                     Registry.LocalMachine.OpenSubKey(
                         string.Format(@"Software\Microsoft\MSBuild\ToolsVersions\{0}", msBuildVersion)))
                 {
-                    if (key == null)
+                    if (key != null)
                     {
-                        return null;
```

Comment:
Does anything check for this returning null? Looks like now it'll return String.Empty if nothing is found.

Reply:
The check is `string.IsNullOrEmpty`on the receiving end, so it shouldn't make any difference.

Reply:
It probably should have been returning `string.Empty` instead of `null` anyway.